src/transformers/models/llava/processing_llava.py

```diff
@@ -52,21 +52,21 @@ class LlavaProcessor(ProcessorMixin):
     """
 
     attributes = ["image_processor", "tokenizer"]
-    valid_kwargs = ["chat_template", "patch_size", "vision_feature_select_strategy", "image_token"]
+    valid_kwargs = ["chat_template", "num_image_tokens", "vision_feature_select_strategy", "image_token"]
     image_processor_class = "AutoImageProcessor"
     tokenizer_class = "AutoTokenizer"
 
     def __init__(
         self,
         image_processor=None,
         tokenizer=None,
-        patch_size=None,
```

Comment:
For backwards compatibility, we can't just remove accepting patch_size immediately. We need to deprecate it and not allow both `patch_size` and `num_image_tokens` to be specified. 

You can use the @deprecate_kwarg decorator to handle this